package.json

```diff
@@ -88,7 +88,7 @@
     "rimraf": "^2.6.2",
     "tslint": "^5.11.0",
     "tslint-config-prettier": "^1.15.0",
-    "typescript": "^3.1.1",
+    "typescript": "^3.5.1",
     "webpack": "^4.5.0",
```

Comment:
Shall we go for TypeScript 3.5.2 since it's out?

Reply:
No problem!